types/react/test/tsx.tsx

```diff
@@ -357,8 +357,14 @@ const Profiler = React.unstable_Profiler;
 </Profiler>;
 
 type ImgProps = React.ComponentProps<'img'>;
-// $ExpectType "async" | "auto" | "sync" | undefined
-type ImgPropsDecoding = ImgProps['decoding'];
+const imgProps: ImgProps = {};
+// the order of the strings in the union seems to vary
+// with the typescript version, so test assignment instead
```

Comment:
Sometimes it's `"async" | "auto" | "sync" | undefined`, sometimes it's `"auto" | "async" | "sync" | undefined`. Not sure why. It doesn't matter for type checking, but dtslint cares about it.